File: matlab/test/Ice/objects/EI.m
% Copyright (c) ZeroC, Inc.

classdef EI < Test.E
    methods
        function obj = EI()
            obj@Test.E(1, 'hello');
        end

        function r = checkValues(obj)
            r = obj.i == 1 && strcmp(obj.s, 'hello');
        end
    end
end

File: csharp/test/Ice/objects/EI.cs
// Copyright (c) ZeroC, Inc.

namespace Ice
{
    namespace objects
    {
        public sealed class EI : Test.E
        {
            public EI() : base(1, "hello")
            {
            }

            public bool checkValues()
            {
                return i == 1 && s == "hello";
            }
        }
    }
}
